src/h5web/providers/api.ts

```diff
@@ -4,10 +4,14 @@ import axios, {
   AxiosResponse,
   CancelTokenSource,
 } from 'axios';
-import { Entity, ProviderError, ValueRequestParams } from './models';
+import {
+  Entity,
+  ProviderError,
+  ValueRequestParams as ValueStoreParams,
+} from './models';
 
 interface ValueRequest {
-  params: ValueRequestParams;
+  params: ValueStoreParams;
```

Comment:
Renamed this type to make the difference with the request query parameters

Reply:
Maybe just add an `s` to match the store's exact name: `ValuesStoreParams`